frontend/src/features/projects/common/interfaces.ts

```diff
@@ -194,6 +194,7 @@ export interface IProject {
   removalFromSplRequestOn?: Date | string;
   removalFromSplApprovedOn?: Date | string;
   removalFromSplRationale?: string;
+  riskId?: number;
```

Comment:
I think if you include `risk` you'll get a string name for each risk.  The `riskId` isn't really useful to the display.